packages/google_sign_in/google_sign_in/test/google_sign_in_test.dart

```diff
@@ -9,6 +9,9 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:google_sign_in/google_sign_in.dart';
 import 'package:google_sign_in/testing.dart';
 import 'package:google_sign_in_platform_interface/google_sign_in_platform_interface.dart';
+import 'package:mocktail/mocktail.dart';
+
+class MockGoogleSignInAccount extends Mock implements GoogleSignInAccount {}
```

Comment:
Please add comments to all of these apparently-unused mocks explaining why they are here, so that someone doesn't remove them as dead code.

Reply:
Thank you for the comment.
Actually, I have a little silly curious about this.
In this part, it just declared unused mock class which only indicates the purpose of "It can be mocked".
I am not sure about this, but Is this necessary?

Reply:
Isn't this the only thing that's validating your change?

Reply:
You are right... Never mind.
I thought this is necessary for a moment but also does not necessary which means it is required for now to pass the PR but it does not need in the future?

Reply:
> it is required for now to pass the PR but it does not need in the future?

GoogleSignInAccount is a reasonable thing to want to be able to mock; having a test that continues to ensure we don't break the ability to mock it (potentially in different ways next time) is useful.

For the mock data classes in in-app purchase you should actually just remove them; nobody should be mocking data classes anyway.

Reply:
I agree.
Removed all mock data classes. Thanks for reviewing.